src/Components/Admins/Form/index.js

```diff
@@ -0,0 +1,132 @@
+import { useState } from 'react';
+import styles from './form.module.css';
+import Input from '../Input';
+import Error from '../Error';
+import Button from '../Button';
+
+function Form() {
+  const [emailValue, setEmailValue] = useState([]);
+  const [passwordValue, setPasswordValue] = useState([]);
+  const [passwordError, setPasswordError] = useState(false);
+  const [emailError, setEmailError] = useState(false);
+  const [save, setSave] = useState(true);
+
+  const onChangeEmailInput = (event) => {
+    setEmailValue(event.target.value);
+  };
+
+  const onChangePasswordInput = (event) => {
+    setPasswordValue(event.target.value);
+  };
+  const onSubmit = (event) => {
+    event.preventDefault();
+    const params = new URLSearchParams(window.location.search);
+    const adminId = params.get('id');
+    let url;
+
+    const options = {
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        email: emailValue,
+        password: passwordValue
+      })
+    };
+
+    if (adminId !== null) {
+      options.method = 'PUT';
+      url = `${process.env.REACT_APP_API}/api/admins/update/${adminId}`;
```

Comment:
If you check on the deployed app in [heroku](https://basd-2021-aq-feature-ms-xltajx.herokuapp.com/admins) , the requests to the server are failing cause `/api` is duplicated. It is probably that `/api` already included in the 
`REACT_APP_API` environment variable. You should remove it from here and add it in your `.env` file

![image](https://user-images.githubusercontent.com/8579539/144353389-32c1e207-a4ee-4277-9171-95eff6e9467c.png)

```suggestion
      url = `${process.env.REACT_APP_API}/admins/update/${adminId}`;
```